documentation/tutorial/08-deploy-worker.md

```diff
@@ -0,0 +1,99 @@
+# Deploy Temporal Worker
+
+This is part of the [Charmed Temporal Tutorial](./00-introduction.md). Please
+refer to this page for more information and the overview of the content.
+
+The [Temporal Worker](https://docs.temporal.io/workers) is the entity which
+listens and polls on specific task queue, and executes code in response to the
+task.
+
+## Deploy
+
+In a production environment, a Temporal worker is deployed in a separate
+environment from the Temporal server. As our server is currently locally
+deployed, we will opt for a direct connection from the worker rather than using
+the ingress.
+
+To deploy Charmed Temporal Worker to a different model, you need to run the
+following commands, which will fetch the charm from
+[Charmhub](https://charmhub.io/temporal-worker-k8s) and deploy it to your model:
+
+```bash
+juju add-model temporal-worker
+juju deploy temporal-worker-k8s
+```
+
+Wait until the application is ready - when it is ready, `juju status` will show:
+
+```
+Model            Controller           Cloud/Region        Version  SLA          Timestamp
+temporal-worker  temporal-controller  microk8s/localhost  3.1.5    unsupported  13:21:49+03:00
+
+App                  Version  Status   Scale  Charm                Channel  Rev  Address         Exposed  Message
+temporal-worker-k8s           waiting      1  temporal-worker-k8s  stable     5  10.152.183.187  no       installing agent
+
+Unit                    Workload  Agent  Address      Ports  Message
+temporal-worker-k8s/0*  blocked   idle   10.1.232.75         Invalid config: wheel-file-name missing
+```
+
+## Configure Worker
+
+In order to test our Temporal worker, we will use a
+[sample resource](https://github.com/canonical/temporal-worker-k8s-operator/tree/main/resource_sample)
+which contains one workflow and one activity. Once downloaded, you can run the
```

Comment:
I'd just add the `git clone` command here instead for simplicity. And then make it clear the poetry command should be run in the `resource_sample` directory